tests/pytests/test_mro_drivers.py

```diff
@@ -235,12 +235,12 @@ def isis_compare_dict():
                                            [0.42064763, 0.18624551, -0.2397057 , 0.85493237],
                                            [0.42065923, 0.18630667, -0.23967382, 0.85492228],
                                            [0.42067144, 0.18636687, -0.23964185, 0.85491211]],
-                           'AngularVelocity': [[-5.797334508173795e-05, 0.0009339642673794059, -0.00011512424986625757],
-                                               [-5.872001786769966e-05, 0.0009343455952639878, -0.00011371124132355403],
-                                               [-6.244619825703372e-05, 0.000935616649435616, -0.00010584418382022431],
-                                               [-5.965469273385757e-05, 0.0009306551165985512, -0.00010670063014175982],
-                                               [-6.001692495046523e-05, 0.0009318842753552843, -0.00010732755544866732],
-                                               [-5.9466333983691536e-05, 0.0009375199811173681, -0.00010690951512741344]]},
+                           'AngularVelocity': [[-0.0006409728984903079, 0.0005054077299115119, 0.0004718267948468069],
+                                               [-0.0006410700774431097, 0.0005044862657976017, 0.0004731836236807216],
+                                               [-0.0006408186407087456, 0.0004992170698116158, 0.0004802237192760833],
+                                               [-0.0006363961683672021, 0.0004989647975959612, 0.00047654664046286975],
+                                               [-0.0006376443791903504, 0.0004996117504290811, 0.00047678850931380653],
+                                               [-0.0006404093657132724, 0.0005028749658176146, 0.0004805228583087444]]},
```

Comment:
We still don't know if these are 100% accurate, so I am okay with how these have been rotated.